community/QClass_2024/Submissions/HW3/Aniekan_Afangideh_HW3_VQE.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>Actually, "Optimal energy: -1.8483884765625" is the same as what we got in class!</p><p>The difference is that the Hamiltonian doesn’t contain nuclear components. You might consider comparing again with what we did in Session 3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>